app/admin-books/webapp/manifest.json

```diff
@@ -11,7 +11,7 @@
         },
         "dataSources": {
             "AdminService": {
-                "uri": "admin/",
+                "uri": "odata/v4/admin/",
```

Comment:
Is this change part of wdi5 v2?

Reply:
This change was necessary because I upgraded to the newest CAP version. This has nothing to do with wdi5 v2.